.github/CONTRIBUTING.md

```diff
@@ -1,112 +1,51 @@
-[fork](https://help.github.com/articles/fork-a-repo/) WooCommerce, commit your changes,
-and [send a pull request](https://help.github.com/articles/using-pull-requests/). We'll be happy to review your changes!
-
-## Feature Requests 🚀
-
-Feature requests can be [submitted to our issue tracker](https://github.com/woocommerce/woocommerce/issues/new?template=Feature_request.md). Be sure to include a description of the expected behavior and use case, and before submitting a request, please search for similar ones in the closed issues.
+There are many ways to contribute to the project!
 
-Feature request issues will remain closed until we see sufficient interest via comments and [👍 reactions](https://help.github.com/articles/about-discussions-in-issues-and-pull-requests/) from the community.
-
-You can see a [list of current feature requests which require votes here](https://github.com/woocommerce/woocommerce/issues?q=label%3A%22votes+needed%22+label%3Aenhancement+sort%3Areactions-%2B1-desc+is%3Aclosed).
-
-## Technical Support / Questions ❓
-
-We don't offer technical support on GitHub so we recommend using the following:
-
-**Reading our documentation**
-Usage docs can be found here: https://docs.woocommerce.com/
-
-If you have a problem, you may want to start with the self help guide here: https://docs.woocommerce.com/document/woocommerce-self-service-guide/
-
-**Technical support for premium extensions or if you're a WooCommerce.com customer**
- from a human being - submit a ticket via the helpdesk
-https://woocommerce.com/contact-us/ 
-
-**General usage and development questions**
-- WooCommerce Slack Community: https://woocommerce.com/community-slack/
-- WordPress.org Forums: https://wordpress.org/support/plugin/woocommerce
-- The WooCommerce Help and Share Facebook group
-
-**Customizations**
-- [WooExperts](https://woocommerce.com/experts/)
-- [Codeable](https://codeable.io/)
+- [Translating strings into your language](https://github.com/woocommerce/woocommerce/wiki/Translating-WooCommerce).
+- Answering questions on the various WooCommerce communities like the [WP.org support forums](https://wordpress.org/support/plugin/woocommerce/).
+- Testing open [issues](https://github.com/woocommerce/woocommerce/issues) or [pull requests](https://github.com/woocommerce/woocommerce/pulls) and sharing your findings in a comment.
+- Testing WooCommerce beta versions and release candidates. Those are announced in the [WooCommerce development blog](https://woocommerce.wordpress.com/).
+- Submitting fixes, improvements, and enhancements.
+- To disclose a security issue to our team, [please submit a report via HackerOne](https://hackerone.com/automattic/).
 
-## Build process 🛠
+If you wish to contribute code, before [forking](https://help.github.com/articles/fork-a-repo/) WooCommerce, commiting your changes,
+and [sendind a pull request](https://help.github.com/articles/using-pull-requests/), please see the information in the sections below.
```

Comment:
Typo: `sending` :) or maybe say `submitting a pull request`. Or maybe switch the sentence around, e.g. something like

`If you wish to contribute code, please read the information in the sections below. Then fork WooCommerce, commit your changes, and submit a pull request 🎉`

Reply:
Thanks, @haszari. I liked your suggestion to change the sentence and pushed a commit to incorporate.